test/performance/trace/xray/agent_config.json

```diff
@@ -0,0 +1,110 @@
+{
+    "agent": {
+        "metrics_collection_interval": 1,
+		"run_as_user": "root",
+        "debug": true
+    },
+    "traces": {
+        "endpoint_override" : "https://127.0.0.1/put-data",
+        "traces_collected": {
+          "xray": {
+          }
+        }
+      },
+      "insecure": true,
+      "metrics": {
+        "namespace": "CloudWatchAgentPerformance",
```

Comment:
Shouldn't this be in a different namespace since the metrics are specific to this test?

Reply:
It wasn't for the other performance tests but I can change it